bin/rails_pg_restore

```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/env ruby
 
 require 'yaml'
+require 'erb'
 require 'optparse'
+require_relative "#{Dir.pwd}/config/environment"
```

Comment:
Yeah, this would be the cleanest way. I'll go ahead and merge though since this PR handles the erb parsing (so it's already a drastic improvement). If you want to clean it up later, feel free to open a PR. Thanks Nate!